src/main/java/uk/gov/companieshouse/uri/web/transformer/CompanyDetailsTransformerImpl.java

```diff
@@ -247,4 +253,14 @@ private String getForeignCompanyRegistryCountry(CompanyProfileApi companyProfile
         }
         return null;
     }
+    
+    private boolean useIncDate(CompanyProfileApi companyProfileApi) {
```

Comment:
Could we call this method useIncorporationDate? I guess Inc is unambiguous in this context but for me at least IncorporationDate is quicker to interpret than IncDate.

Reply:
Method name now updated to be useIncorporationDate